### ISSUE TITLE
Enable Support for Multiple Jira Project Keys

### ISSUE THREAD

**Issue by SaachiNayyer** [AxisCommunications/backstage-plugins#232]
### Describe the feature

Currently, the API only processes the first project key when handling a request with multiple Jira project keys. This limitation means that if an entity contains multiple project keys (e.g., abc, def), only data related to the first key is fetched and returned in the response.

### Added value

This enhancement would be highly beneficial for users managing multiple Jira projects associated with a single entity

### Additional context

assume a annotation 
annotations:
  jira.com/project-key: abc,def,ghi
with current implementation query created is project in (abc)....
expected query should be 
project in (abc,def,ghi)....

**Comment by anicke** [comment]
Yes, that sounds reasonable. I don't know if there is a reason why it current only uses the first project set.

**Comment by anicke** [comment]
Fixed in #233?
